addon/modifiers/enable-dropdown.js

```diff
@@ -1,8 +1,10 @@
-import { setModifierManager } from '@ember/modifier';
+import { setModifierManager, capabilities } from '@ember/modifier';
 import $ from 'jquery';
```

Comment:
```suggestion
import jQuery from 'jquery';
```
You forget a jquery here